### PR TITLE
Add more details on release channels

### PR DESCRIPTION
**Pull request by justindeguzman**

### CODE REVIEW

File: docs/en/cloud/manage/upgrades.md [hunk 1]
```diff
@@ -17,6 +17,19 @@ At times, it may be necessary for us to update your service, which could require
...
+
+- Receive the latest ClickHouse releases
+- More frequent upgrades as new releases are tested
+- Recommended for non-production environments
+- Supported for both the **Production** and **Development** tiers
+
+## Default release channel
+
+- Recommended for production environments
+- New minor versions are released at least two weeks after the **Fast release** channel
```
Comment (Crazybus) [inline docs/en/cloud/manage/upgrades.md:34]: @nikitamikhaylov two weeks is a place holder. What do you think is a reasonable amount of time between the fast and regular release channels for new releases?

**Review by Crazybus — COMMENTED** [review]
Wording looks great 👍 

One thing to add still would be a disclaimer about some changes going out faster as needed for things like security: https://github.com/ClickHouse/docs-private/issues/71#issuecomment-2112980905.

File: docs/en/cloud/manage/upgrades.md [hunk 2]
```diff
@@ -17,6 +17,19 @@ At times, it may be necessary for us to update your service, which could require
...
+You are able to specify the upgrade schedule for your ClickHouse Cloud service by subscribing to a specific release channel. Besides the default upgrade schedule, we offer a **Fast release** channel if you would like your services to receive updates ahead of the default release schedule. Subscribing to the **Fast release** channel for early upgrades is recommended only for non-production environments and can be requested by logging a support ticket.
+
+### Fast release channel (early upgrades)
+
+- Receive the latest ClickHouse releases
+- More frequent upgrades as new releases are tested
+- Recommended for non-production environments
+- Supported for both the **Production** and **Development** tiers
+
+## Default release channel
```
Comment (Crazybus) [inline docs/en/cloud/manage/upgrades.md:31]:
```suggestion
## Regular release channel
```

I'm not sure if this is true for external facing documentation too, but at least internally we decided to rename from default to regular in https://github.com/ClickHouse/data-plane-application/issues/8859

**Comment by justindeguzman** [comment]
I'm just going to merge this for now–if `New minor versions are released at least two weeks after the **Fast release** channel` needs to be changed let me know.

**Pull request merged**